rust/sdk/src/arb.rs

```diff
@@ -1,3 +1,7 @@
+// Used by trees::arb
+// Can't really be moved to `trees` since Rust disallows
+// implementing foreign traits on foreign types
```

Comment:
yeah, this is a peculiarity of the Rust way of doing things, which leads to good libraries implementing Arbitrary even though they don’t use it internally